apps/what-day/src/index.ts

```diff
@@ -0,0 +1,14 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-search-apps/what-day",
+  name: "WhatDay",
+  description: "Shows what day is on a particular date",
+  queryToData,
+  Component,
+  // screenshotPath: "./files/screenshot.png",
+  exampleSearchQueries: ["what day is it", "what day is 01/01/2000", "what day is it tomorrow"],
```

Comment:
The query "what day it is" doesn't work, I guess you would have to add a condition like you did with "tomorrow", "today" and "yesterday"

Reply:
adding "what day is it" seems to pose a problem as when the query is "what day is it 15/01/2012" it will show wrong result.

Reply:
Understandable, let's remove it then.

Reply:
yes i have removed it